pages/linux/scontrol.md

```diff
@@ -0,0 +1,24 @@
+# scontrol
+
+> View information about and modify jobs.
+> More information: <https://slurm.schedmd.com/scontrol.html>.
+
+- Show information for job:
+
+`scontrol show job {{job_id}}`
+
+- Suspend a comma-separated list of running job:
+
+`scontrol suspend {{job_id}}`
+
+- Resume a suspended job:
+
+`scontrol resume {{job_id}}`
+
+- Hold a queued job (gives it least priority):
```

Comment:
The docs doesn't talk about `priority`, said `Prevent a job from being started`.
You said `give it least priority`, so can the job start if it's the last is the queue?
```suggestion
- Hold a comma-separated list of queued jobs (give it least priority):
```
or
```suggestion
- Hold a comma-separated list of queued jobs (Use `release` command to permit the jobs to be scheduled):
```